[2024-09-07T13:03:11.321+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-07T13:03:11.365+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_etl.load_data_to_warehouse manual__2024-09-07T13:03:08.599435+00:00 [queued]>
[2024-09-07T13:03:11.375+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_etl.load_data_to_warehouse manual__2024-09-07T13:03:08.599435+00:00 [queued]>
[2024-09-07T13:03:11.376+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-09-07T13:03:11.390+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data_to_warehouse> on 2024-09-07 13:03:08.599435+00:00
[2024-09-07T13:03:11.397+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2025) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-07T13:03:11.398+0000] {standard_task_runner.py:63} INFO - Started process 2026 to run task
[2024-09-07T13:03:11.398+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_etl', 'load_data_to_warehouse', 'manual__2024-09-07T13:03:08.599435+00:00', '--job-id', '626', '--raw', '--subdir', 'DAGS_FOLDER/reddit.py', '--cfg-path', '/tmp/tmpf1wg26n7']
[2024-09-07T13:03:11.400+0000] {standard_task_runner.py:91} INFO - Job 626: Subtask load_data_to_warehouse
[2024-09-07T13:03:11.464+0000] {task_command.py:426} INFO - Running <TaskInstance: reddit_etl.load_data_to_warehouse manual__2024-09-07T13:03:08.599435+00:00 [running]> on host 42d604ea5ce6
[2024-09-07T13:03:11.586+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='reddithackathon' AIRFLOW_CTX_DAG_ID='reddit_etl' AIRFLOW_CTX_TASK_ID='load_data_to_warehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T13:03:08.599435+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-07T13:03:08.599435+00:00'
[2024-09-07T13:03:11.587+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-07T13:03:11.625+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.625+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.626+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.626+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.626+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.627+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.627+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.627+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.628+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.628+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.628+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.628+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.629+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.629+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.629+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.629+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.630+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.630+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.630+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.630+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.631+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.631+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.631+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.631+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.632+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.632+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.632+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.632+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.633+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.633+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.634+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.634+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.635+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.635+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.636+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.636+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.637+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.637+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.637+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.638+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.638+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.639+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.639+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.639+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.640+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.640+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.640+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.641+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.641+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.641+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.641+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.642+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.642+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.642+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.642+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.643+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.643+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.643+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.644+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.644+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.644+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.644+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.645+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.645+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.645+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.645+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.646+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.646+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.646+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.646+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.647+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.647+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.647+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.647+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.647+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.648+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.648+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.648+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.648+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.649+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.649+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.649+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.650+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.650+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.650+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.651+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.651+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.651+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.652+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.652+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.653+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.653+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.654+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.654+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.654+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.655+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.655+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.655+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.656+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.656+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.656+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.656+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.657+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.657+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.657+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.658+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.658+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.658+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.659+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.659+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.659+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.659+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.660+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.660+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.660+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.660+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.661+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.661+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.661+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.661+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.662+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.662+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.662+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.663+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.663+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.663+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.663+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.664+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.664+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.664+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.664+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.665+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.665+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.665+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.666+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.666+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.666+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.667+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.667+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.668+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.668+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.669+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.669+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.670+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.670+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.671+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.671+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.672+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.672+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.672+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.673+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.673+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.673+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.673+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.674+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.674+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.674+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.674+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.675+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.675+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.675+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.676+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.676+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.676+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.676+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.677+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.677+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.677+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.677+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.678+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.678+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.678+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.679+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.679+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.679+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.680+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.680+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.680+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.680+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.681+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.681+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.681+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.682+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.682+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.682+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.683+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.683+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.683+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.684+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.684+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.685+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.685+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.686+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.686+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.687+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.687+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.687+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.688+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.688+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.688+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.689+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.689+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.689+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.689+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.690+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.690+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.690+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.690+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.691+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.691+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.691+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.691+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.692+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.692+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.692+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.693+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.693+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.693+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.693+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.694+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.694+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.694+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.694+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.695+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.695+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.695+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.695+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.696+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.696+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.696+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.696+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.697+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.697+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.697+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.698+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.698+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.698+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.698+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.699+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.699+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.699+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.700+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.700+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.700+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.701+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.701+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.701+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.702+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.702+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.703+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.703+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.704+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.704+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.704+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.705+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.705+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.706+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.706+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.706+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.706+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.707+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.707+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.707+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.707+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.708+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.708+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.708+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.709+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.709+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.709+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.709+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.710+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.710+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.710+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.710+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.711+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.711+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.711+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.711+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.712+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.712+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.712+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.712+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.713+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.713+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.713+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.713+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.714+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.714+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.714+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.714+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.715+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.715+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.715+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.715+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.716+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.716+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.716+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.717+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.717+0000] {logging_mixin.py:188} INFO - Error loading data: connection to server on socket "/var/run/postgresql/.s.PGSQL.54322" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
[2024-09-07T13:03:11.718+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-07T13:03:11.718+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-07T13:03:11.734+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=reddit_etl, task_id=load_data_to_warehouse, execution_date=20240907T130308, start_date=20240907T130311, end_date=20240907T130311
[2024-09-07T13:03:11.772+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-07T13:03:11.795+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-07T13:03:11.796+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
